app/js/newsfeed.js

```diff
@@ -119,7 +119,7 @@ module.exports = {};
    * @returns {Boolean} Always returns true.
    */
   function displayNews(posts) {
-    loadingDots.classList.add("hidden");
+    loadingDots.parentElement.removeChild(loadingDots);
```

Comment:
@le717 sorry I hadn't realised `.remove()` was an actual method! Just committed again using `remove()` which does indeed work. I'd suggest squash merging this particular branch when the time comes.